packages/trpc/server/routers/viewer/teams/inviteMember/utils.ts

```diff
@@ -227,22 +227,29 @@ export async function createNewUsersConnectToOrgIfExists({
 }) {
   // fail if we have invalid emails
   usernamesOrEmails.forEach((usernameOrEmail) => checkInputEmailIsValid(usernameOrEmail));
-
   // from this point we know usernamesOrEmails contains only emails
   await prisma.$transaction(
     async (tx) => {
       for (let index = 0; index < usernamesOrEmails.length; index++) {
         const usernameOrEmail = usernamesOrEmails[index];
+        // Weird but orgId is defined only if the invited user email matches orgAutoAcceptEmail
         const { orgId, autoAccept } = connectionInfoMap[usernameOrEmail];
         const [emailUser, emailDomain] = usernameOrEmail.split("@");
-        const username =
+
+        // An org member can't change username during signup, so we set the username
+        const orgMemberUsername =
           emailDomain === autoAcceptEmailDomain
             ? slugify(emailUser)
             : slugify(`${emailUser}-${emailDomain.split(".")[0]}`);
 
+        // As a regular team member is allowed to change username during signup, we don't set any username for him
+        const regularTeamMemberUsername = null;
```

Comment:
I think we accidentally started setting it for regular teams by this PR
https://github.com/calcom/cal.com/pull/11238/files
cc @joeauyeung for confirmation